app/scripts/lib/experiments/choices/sentry.js

```diff
@@ -0,0 +1,33 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+define((require, exports, module) => {
+  'use strict';
+
+  const BaseExperiment = require('./base');
+
+  module.exports = class SentryExperiment extends BaseExperiment {
```

Comment:
nit: This experiment is missing a bit of docs. Do you think someone new to the project will be able to understand what this does? I guess they can look at the `sentryEnabled` code too.